### PR TITLE
client: add toxics and promote from wip

### PR DESCRIPTION
**Pull request by Sirupsen**
@eapache @xthexder

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# toxiproxy-go
+
+This is the Go client library for the
+[Toxiproxy](https://github.com/shopify/toxiproxy) API. Please read the [usage
+section in the Toxiproxy README](https://github.com/shopify/toxiproxy#usage)
+before attempting to use the client.
+
+For Usage please see the Godoc documentation for the package.
```
Comment (xthexder) [inline client/README.md]: It woud be good to actually provide a link to the godocs.

File: client/client.go [hunk 2]
```diff
@@ -46,6 +56,8 @@ func (client *Client) Proxies() (map[string]*Proxy, error) {
 	return proxies, nil
 }
 
+// NewProxy instantiates a new proxy instance. Note Create() must be called on
+// it to create it.
```
Comment (xthexder) [inline client/client.go]: This should probably note that the proxy is not enabled by default

**Comment by xthexder** [comment]
Changes look good :+1: 

I noticed while reading through the main Readme again that the Curl example is out of date (it doesn't contain any of the proxy enabled/disabled stuff)
It would probably be a good idea to revisit that at some point.

**Comment by Sirupsen** [comment]
Rebased and fixed your comments. Fixed the main README change in  https://github.com/Shopify/toxiproxy/commit/b979c77411aa6336d716a1f074985248a9e56cb2

File: client/client.go [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// The Toxiproxy Go client provides a wrapper around the Toxiproxy HTTP API for
```
Comment (eapache) [inline client/client.go]: By convention, package-level godoc starts with "Package $NAME"

File: client/client.go [hunk 4]
```diff
@@ -151,6 +171,29 @@ func (proxy *Proxy) Toxics(direction string) (map[string]interface{}, error) {
 	return toxics, nil
 }
 
+// SetToxic sets the parameters for a toxic with a given name in the direction.
+// See https://github.com/Shopify/toxiproxy#toxics for a list of all Toxics.
+func (proxy *Proxy) SetToxic(name string, direction string, fields Fields) (map[string]interface{}, error) {
```
Comment (eapache) [inline client/client.go]: should direction (and maybe name) be enum types rather than arbitrary strings?

**Comment by eapache** [comment]
A few minor comments, but :+1:

**Pull request merged**